trade-finance-manager-ui/server/api.js

```diff
@@ -421,6 +421,29 @@ const login = async (username, password) => {
   }
 };
 
+/**
+ *
+ * @param {import('@ukef/dtfs2-common').handleSsoRedirectFormRequest} handleSsoRedirectFormRequest
+ * @returns {Promise<import('@ukef/dtfs2-common').handleSsoRedirectFormResponse>}
+ */
+const handleSsoRedirectForm = async (handleSsoRedirectFormRequest) => {
+  try {
+    const response = await axios({
+      method: 'post',
```

Comment:
Keeping this as `post` as it's technically not idempotent due to issuing a token every time it is hit